### PR TITLE
List supported releases

### PR DESCRIPTION
**Pull request by jacob-delgado**
Based on istio discussion about fixes.

List when they were released and when their EOL is.

Also list if a version contains a CVE.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by therealmitchconnors** [comment]
This looks great, and should be coupled with 1) strong source controls over this file, and 2) an istioctl command that checks this page to report support/cve status.

**Comment by brian-avery** [comment]
This looks great.

**Comment by brian-avery** [comment]
Should this list the next scheduled release?

File: content/en/about/supported-releases/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+The status of currently supported releases.
+
+## Supported versions of Istio
+
+|     | Curently Supported | Release Date    | End of Life       |
+|-----|--------------------|-----------------|-------------------|
+| 1.7 | Yes                | August 21, 2020 |                   |
+| 1.6 | Yes                | May 21, 2020    | November 21, 2020 |
+| 1.5 | No                 |                 |                   |
+| 1.4 | No                 |                 |                   |
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:17]: 1.5 and earlier No?

File: content/en/about/supported-releases/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Contains known CVEs
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:19]: How should a customer use this? I'm assuming by design any release will have a CVE in few months. Should we add the opposite i.e. releases without Known CVEs?

Comment (jacob-delgado) [reply content/en/about/supported-releases/index.md]: I like your way better. Changed.

File: content/en/about/supported-releases/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/about/supported-releases/index.md]: Changed

**Comment by jacob-delgado** [comment]
> This looks great, and should be coupled with 1) strong source controls over this file, and 2) an istioctl command that checks this page to report support/cve status.

Added this as a feature request https://github.com/istio/istio/issues/28234. This falls outside of the scope of this, for now imo, but it is work that should be done.



> Should this list the next scheduled release?

I think this would have to be part of the release managers supported announcement checklist. They know the most about what goes in a release, EOL, etc...

**Comment by nrjpoddar** [comment]
This is looking good.

What does "Supported Releases" mean? That's missing from this page. Any user facing document like this should explain the terminologies to remove ambiguity. Something like this might be helpful:

Supported releases of Istio include releases that are in the active maintenance window and are patched for security and bug fixes. No backward incompatible changes are introduced in patch versions of these releases.

File: content/en/about/supported-releases/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Supported versions of Istio
+
+|                 | Currently Supported | Release Date    | End of Life       |
+|-----------------|---------------------|-----------------|-------------------|
+| 1.7             | Yes                 | August 21, 2020 |                   |
+| 1.6             | Yes                 | May 21, 2020    | November 21, 2020 |
+| 1.5 and earlier | No                  |                 |                   |
+
+### Releases without known CVEs
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:18]: Should we expand CVEs here?

File: content/en/about/supported-releases/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+|                 | Releases with no known CVEs |
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:20]:
```suggestion
|  LTS Release               |  Patched versions with no known CVEs |
```

File: content/en/about/supported-releases/index.md (hunk 3 above)
Comment (jacob-delgado) [reply content/en/about/supported-releases/index.md]: done

**Comment by jacob-delgado** [comment]
cc @frankbu finding a good home for this is important from a UX perspective as well. Any thoughts on where to place this? Should I move this into the `release-cadence` page instead of creating a new directory?

File: content/en/about/supported-releases/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Supported Releases
+description: The current supported Istio releases.
+weight: 35
+icon: cadence
+---
+
+The status of currently supported releases. Supported releases of Istio include releases that are in the active
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:8]:
```suggestion
This page lists the status and timelines of currently supported releases. Supported releases of Istio include releases that are in the active
```

File: content/en/about/supported-releases/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+maintenance window and are patched for security and bug fixes. Patch releases do not contain backward incompatible
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:9]:
```suggestion
maintenance window and are patched for security and bug fixes. Subsequent patch releases on a LTS release do not contain backward incompatible
```

File: content/en/about/supported-releases/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+changes from the same minor release. For more information refer to the [Istio support policy](../release-cadence/).
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:10]:
```suggestion
changes. For more information refer to the [Istio support policy](../release-cadence/).
```

File: content/en/about/supported-releases/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+## Supported versions of Istio
+
+|                 | Currently Supported   | Release Date    | End of Life       |
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:14]:
```suggestion
|Version         | Currently Supported   | Release Date    | End of Life       |
```

File: content/en/about/supported-releases/index.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+description: The current supported Istio releases.
+weight: 35
+icon: cadence
+---
+
+The status of currently supported releases. Supported releases of Istio include releases that are in the active
+maintenance window and are patched for security and bug fixes. Patch releases do not contain backward incompatible
+changes from the same minor release. For more information refer to the [Istio support policy](../release-cadence/).
+
+## Supported versions of Istio
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:12]:
```suggestion
## Support status of Istio releases
```

**Comment by frankbu** [comment]
> cc @frankbu finding a good home for this is important from a UX perspective as well. Any thoughts on where to place this? 

@jacob-delgado Where you have it now LGTM.

**Comment by jacob-delgado** [comment]
cc @nrjpoddar Need your consent and another review

File: content/en/about/supported-releases/index.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+icon: cadence
+---
+
+This page lists the status and timelines for currently supported releases. Supported releases of Istio include releases that are in the active
+maintenance window and are patched for security and bug fixes. Subsequent patch releases on a LTS release do not contain backward incompatible
+changes. For more information refer to the [Istio support policy](../release-cadence/).
+
+## Support status of Istio releases
+
+|Version         | Currently Supported   | Release Date    | End of Life       |
```
Comment (jacob-delgado) [inline content/en/about/supported-releases/index.md:14]:
```suggestion
| Version         | Currently Supported   | Release Date    | End of Life       |
```

File: content/en/about/supported-releases/index.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+This page lists the status and timelines for currently supported releases. Supported releases of Istio include releases that are in the active
+maintenance window and are patched for security and bug fixes. Subsequent patch releases on a LTS release do not contain backward incompatible
+changes. For more information refer to the [Istio support policy](../release-cadence/).
+
+## Support status of Istio releases
+
+| Version         | Currently Supported   | Release Date    | End of Life       |
+|-----------------|-----------------------|-----------------|-------------------|
+| master          | No, development only  |                 |                   |
+| 1.8             | Not yet.              | ETA Nov 2020    |                   |
```
Comment (nrjpoddar) [inline content/en/about/supported-releases/index.md:17]:
```suggestion
| 1.8             | Not yet              | ETA Nov 2020    |                   |
```

Comment (nrjpoddar) [reply content/en/about/supported-releases/index.md:17]: May be we can delete the 1.8 i.e. future looking version row altogether?

**Comment by nrjpoddar** [comment]
@googlebot I fixed it.

**Comment by nrjpoddar** [comment]
If the CLA's still complaining, squash my commits and rebase.

**Review by frankbu — APPROVED** [review]

**Pull request merged**